lib/common/util_benchmark_test.go

```diff
@@ -0,0 +1,170 @@
+package sebakcommon
```

Comment:
I think any kind of benchmark should be in its own source tree and run on demand.
We should not have benchmark as part of the unit tests.

Reply:
@Geod24 Is that common rule or just your think? It's just curious because I'm not familiar with benchmark test :)
As you know, the benchmark tests are not running when you didn't include '-bench' option.